examples/synthesis_json.py

```diff
@@ -69,7 +72,6 @@ def extract_data(files, feature, params=None):
     try:
```

Comment:
There is a problem with this bit of code, which has prevented me from fitting trunk_radius.